lib/python/picongpu/plugins/phase_space.py

```diff
@@ -188,7 +188,8 @@ def get_iterations(self, species, species_filter='all', ps=None):
 
         return iterations
 
-    def get(self, species, species_filter='all', ps=None, iteration=None):
+    def get(self, species=None, species_filter='all', iteration=None, ps=None,
```

Comment:
See above

Reply:
Is it still okay to change the order of the variables here? I recall you wanted to have species, filter and iteration before all other class dependent options so that is why I made the change.

Reply:
Good question here. Well, ok.

If you change it, please adjust the `.rst` docs in `docs/source/usage/plugins/phaseSpace.rst` as well.